src/khoj/routers/api_chat.py

```diff
@@ -752,13 +758,18 @@ async def chat(
     await is_ready_to_chat(user)
     conversation_commands = [get_conversation_command(query=q, any_references=True)]
 
+    _custom_filters = ""
     if conversation_commands == [ConversationCommand.Help]:
-        conversation_config = await ConversationAdapters.aget_user_conversation_config(user)
-        if conversation_config == None:
-            conversation_config = await ConversationAdapters.aget_default_conversation_config()
-        model_type = conversation_config.model_type
-        formatted_help = help_message.format(model=model_type, version=state.khoj_version, device=get_device())
-        return StreamingResponse(iter([formatted_help]), media_type="text/event-stream", status_code=200)
+        if not q:
```

Comment:
Would this work? Wouldn't `q` at least contain `/help` at this point? You can test it by disable the websocket in the code in the client.